### PR TITLE
Small fixes and improvements

### PR DESCRIPTION
**Pull request by osrecio**
Add FileSystem to manage Files
Fix Construct Controllers Path
Delete Object Manager
Fix PHPCS and PHPMD Problems

@wiardvanrij Check all functionality before Merge. I checked all I know but you have more know-how about your module than me.

### CODE REVIEW

**Comment by wiardvanrij** [comment]
Thank you, I will test & check this soon :)

File: Console/Command/Clear.php [hunk 1]
```diff
@@ -98,36 +96,51 @@ private function clearRequest()
...
+     * @return bool|string
+     */
+    private function allowFlush()
     {
-        $path = $this->filesystem->getDirectoryRead(DirectoryList::VAR_DIR)->getAbsolutePath();
-        $file = $path . '/allow-opcache.flush';
-
-        if (file_exists($file)) {
-            unlink($file);
+        $fileName = 'allow-opcache.flush';
```
Comment (wiardvanrij) [inline Console/Command/Clear.php]: can you make a const of this?

File: Console/Command/Clear.php [hunk 2]
```diff
@@ -5,63 +5,61 @@
...
     }
 
     /**
-     * @param \Symfony\Component\Console\Input\InputInterface $input
-     * @param \Symfony\Component\Console\Output\OutputInterface $output
-     *
-     * @return \Symfony\Component\Console\Output\OutputInterface|null
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     * @return int|null|void
```
Comment (JKetelaar) [inline Console/Command/Clear.php]: It returns nothing (void)

File: Console/Command/Clear.php [hunk 3]
```diff
@@ -5,63 +5,61 @@
...
 {
 
     /**
      * @var StoreManagerInterface
      */
-    protected $storeManager;
+    private $storeManager;
 
     /**
      * @var \Magento\Framework\Filesystem
```
Comment (JKetelaar) [inline Console/Command/Clear.php]:
```
* @var Filesystem
```

File: Console/Command/Clear.php (hunk 1 above)
Comment (osrecio) [reply Console/Command/Clear.php]: Yes I will change it for constant

File: Console/Command/Clear.php (hunk 3 above)
Comment (osrecio) [reply Console/Command/Clear.php]: I will change it 👍

File: Console/Command/Clear.php (hunk 2 above)
Comment (osrecio) [reply Console/Command/Clear.php]: I will apply 
```
/**
     * {@inheritdoc}
     */
```

**Comment by osrecio** [comment]
@JKetelaar @wiardvanrij Thanks for your suggestions. I have fixed what you told me.

**Comment by wiardvanrij** [comment]
@osrecio thank you very much 👍

**Pull request merged**